manifest_staging/charts/csi-secrets-store-provider-azure/config/fluentd.conf

```diff
@@ -0,0 +1,55 @@
+@include kubernetes.conf
+
+# Retag to prefix driver container events with akvssd
+<match kubernetes.var.log.containers.secrets-store-csi-driver**secrets-store**.log>
```

Comment:
Does this mean we'll only get logs for `secrets-store` container? 

If _yes_, we should also collect logs from `node-driver-registrar` and `liveness-probe` containers to debug driver registration errors.

Reply:
Yes. I wanted to add this, but json logging for these containers is not supported. Aren't you following up on this?

Reply:
The work is planned here: https://github.com/orgs/kubernetes-csi/projects/46 but it'll be a while before it's supported

Reply:
Correct. So once we have that support we can come back and enable this. Thoughts?

Reply:
As discussed, let's enable this when we get json support as it's needs to send data to Geneva.